src/app/tests/suites/certification/Test_TC_IDM_1_2.yaml

```diff
@@ -20,342 +19,366 @@ PICS:
 
 config:
     nodeId: 0x12344321
-    cluster: "Basic Information"
-    endpoint: 0
+    InvokeRequestMessage.Cluster: 0x00000006 # OnOff
+    InvokeRequestMessage.Command: 0x00000000 # Off
+    InvokeRequestMessage.EndPoint: 1
+    InvokeRequestMessage.Payload: {}
 
 tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    #### Test Setup
+
+    - label: "Get the list of available endpoints on the device"
+      cluster: "Descriptor"
+      command: "readAttribute"
+      attribute: "PartsList"
+      endpoint: "*"
```

Comment:
If we are just getting the list of all endpoints then:
1. Shouldn't we just do this on EP0?
2. 0 will not be in the list, right?

It looks like GetUnsupportedEndpoint includes the "response endpoints" as being "supported", which is why this work, but it's a bit non-obvious and maybe worth documenting.  And we should definitely do this read only on EP0...